src/main/app/initialize.ts

```diff
@@ -235,7 +235,7 @@ function initializeBeforeAppReady() {
     AllowProtocolDialog.init();
 
     if (isDev && process.env.NODE_ENV !== 'test') {
-        log.info('In development mode, deeplinking is disabled');
+        app.setAsDefaultProtocolClient('mattermost-dev');
```

Comment:
Shouldn't we keep the log, or is it no longer valid?

Reply:
It's no longer valid, deep linking will technically be enabled in development mode, just under `mattermost-dev:`